Index: kscreen-5.3.2/tests/kded/CMakeLists.txt
===================================================================
--- kscreen-5.3.2.orig/tests/kded/CMakeLists.txt
+++ kscreen-5.3.2/tests/kded/CMakeLists.txt
@@ -25,6 +25,6 @@ macro(ADD_KDED_TEST testname)
     ecm_mark_as_test(${testname})
 endmacro()
 
-add_kded_test(testgenerator)
+#add_kded_test(testgenerator)
 add_kded_test(serializertest)
 #add_kded_test(testdaemon)
